outputs.tf

```diff
@@ -1,11 +1,13 @@
 output "linux_function_app" {
   description = "Linux Function App output object if Linux is chosen. Please refer to `./modules/linux-function/README.md`"
   value       = try(module.linux_function["enabled"], null)
+  sensitive   = true
```

Comment:
Hum no, I fully disagree with this.
It will prevent using correctly the module and access properties in a stack context with Terraform or OpenTofu.